env0/utils.go

```diff
@@ -50,6 +50,29 @@ func stringInSlice(str string, strs []string) bool {
 	return false
 }
 
+func reasourceDataGetValue(fieldName string, d *schema.ResourceData) interface{} {
```

Comment:
The way terraform sdk handles `Get` is clunky.
I'll explain in the next few lines....